src/app/lib/utilities/variantHandler/index.js

```diff
@@ -5,9 +5,10 @@ export const servicesWithVariants = {
   serbian: ['lat', 'cyr'],
   ukchina: ['simp', 'trad'],
   zhongwen: ['simp', 'trad'],
+  ukrainian: ['default', 'ru'],
```

Comment:
I like this approach of considering it as a variant of Ukrainian 👍. 
Just wondering what would be more understandable, either using `default` or `uk`

Reply:
I have it as `ru` variant because this is the value we pass in from the App.jsx. And we get `ru` from the metadata lang value

Reply:
Yeah, I meant the other bit: `ukrainian: ['uk', 'ru']` instead of `ukrainian: ['default', 'ru']`.